### PR TITLE
Feat/edit project page

### PR DESCRIPTION
**Pull request by kubovych**

### CODE REVIEW

File: backend/src/data/mappers/mapPrivileges.ts [hunk 1]
```diff
@@ -1,20 +1,19 @@
-import { Privilege, PrivilegeWithBakersAmount } from './../../common/types/project';
+import { Privilege, PrivilegeWithBakersAmount } from '../../common/types/project';
+
 const mapPrivileges = (
   privileges: Privilege[],
   bakersDonation: number[]
```
Comment (yevhenii-moroziuk) [inline backend/src/data/mappers/mapPrivileges.ts]: If bakersDonation can be null, than use bakersDonation: number[] | null

File: backend/src/services/faq/faq.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+}
+export default class FAQServise {
+  readonly #faqRepository: FAQRepository;
+
+  constructor(faqRepository: FAQRepository) {
+    this.#faqRepository = faqRepository;
+  }
+
+  public async save(faqs:FAQ[]) {
+    const tags = await this.#faqRepository.save(faqs);
```
Comment (yevhenii-moroziuk) [inline backend/src/services/faq/faq.ts]: You can return a promise: 
`return this.#faqRepository.save(faqs);`

File: backend/src/services/project/project.ts [hunk 3]
```diff
@@ -77,6 +89,12 @@ export default class ProjectService {
     return project; // rewrite when error handling middleware works
   }
 
+  public async getForEdit(id: string) {
+    const project: CreateProject = await this.#projectRepository.getForEdit(id);
+    console.log(project);
```
Comment (yevhenii-moroziuk) [inline backend/src/services/project/project.ts]: log

File: backend/src/services/projectPrivilege/projectPrivilege.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
+// import { DonatorsPrivilege } from '../../data/entities';
```
Comment (yevhenii-moroziuk) [inline backend/src/services/projectPrivilege/projectPrivilege.ts]: comment

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

**Review by maksim-luzin — APPROVED** [review]

**Review by yevhenii-moroziuk — APPROVED** [review]

**Pull request merged**